### PR TITLE
Nodejs week3

### PR DESCRIPTION
**Pull request by melo4yxa77796**

### CODE REVIEW

File: nodejs-week3/app.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    if (orderBy.length === 2) {
+      const [field, order] = orderBy;
+
+      if (
+        allowedSortFields.includes(field) &&
+        allowedSortOrders.includes(order.toUpperCase())
+      ) {
+        query = query.orderBy(field, order);
+      } else {
+        return res.status(400).json({ error: "Invalid sort parameter" });
```
Comment (kralle333) [inline nodejs-week3/app.js:45]: here you can give different errors depending on what part of the request was wrong ;)

File: nodejs-week3/app.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  }
+
+  console.log("SQL Query:", query.toSQL().sql);
+
+  try {
+    const data = await query;
+    res.json({ data });
+  } catch (e) {
+    console.error("Database query error:", e);
+    res.status(500).json({ error: "Internal server error" });
```
Comment (kralle333) [inline nodejs-week3/app.js]: No need for writing Internal server error here, it is already in the error code 500 ;)

**Review by kralle333 — APPROVED** [review]
Good job!

Only have a small comment, but it isn't necessary to implement if you dont have time

File: nodejs-week3/app.js (hunk 2 above)
Comment (melo4yxa77796) [reply nodejs-week3/app.js]: i made changes

File: nodejs-week3/app.js (hunk 1 above)
Comment (melo4yxa77796) [reply nodejs-week3/app.js:45]: i will try later

**Pull request merged**